crates/holochain_sqlite/src/sql/cell/schema/2.sql

```diff
@@ -0,0 +1,168 @@
+-- no-sql-format --
```

Comment:
: (

Reply:
Have you tried sql formatting for an actual mutli-query script? It's terrible. No whitespace allowed in between queries.

Reply:
Not worth spending time now, but I'd like to have a formatter, even if we have to pick a different one, or add comment just for spacing or something.